Makefile

```diff
@@ -7,11 +7,18 @@ GOPATH := $(shell go env GOPATH)
 GOARCH := $(shell go env GOARCH)
 GOOS := $(shell go env GOOS)
 
+CRD_GEN_PATH=operator-kustomize/crds/
+OPERATOR_YAML=./operator-kustomize/
+TENANT_CRD=$(OPERATOR_YAML)crds/minio.min.io_tenants.yaml
+PLUGIN_HOME=./kubectl-minio
+PLUGIN_OPERATOR_YAML=$(PLUGIN_HOME)/static
+PLUGIN_CRD_COPY=$(PLUGIN_OPERATOR_YAML)/crd.yaml
 all: build
 
 getdeps:
 	@mkdir -p ${GOPATH}/bin
 	@which golangci-lint 1>/dev/null || (echo "Installing golangci-lint" && curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin v1.27.0)
+	@go get sigs.k8s.io/controller-tools/cmd/controller-gen
```

Comment:
Since we use go.mod lets avoid doing `go get` right here. because this will in-advertently update `go.mod` we should perhaps download the released binaries if any.

@nitisht @kerneltime

Reply:
Nit: This should ideally be done via a Docker container to lock the version of controller-gen. Right now every developer uses a different version from what I can tell.

Reply:
Removed go get or now. We can look at fixing in a separate PR